test/integration/middleware/with-builtin-module/test/index.test.js

```diff
@@ -22,14 +23,16 @@ describe('Middleware importing Node.js built-in module', () => {
     return `Module not found: Can't resolve '${name}'`
   }
 
-  function getBuiltinApisNotSupported(name) {
-    return `Native Node.js APIs are not supported in the Edge Runtime. Found \`${name}\` imported.\n`
+  function escapeLF(s) {
+    return s.replace(/\n/g, '\\n')
   }
 
   describe('dev mode', () => {
     let output = ''
 
-    beforeAll(async () => {
+    // restart the app for every test since the latest error is not shown sometimes
```

Comment:
This sounds like a bug, was this the case before these changes?

Reply:
Sounds good to investigate separately x-ref: https://github.com/vercel/next.js/issues/36575